server/http/dialog/install.go

```diff
@@ -47,25 +47,25 @@ func NewInstallAppDialog(m *apps.Manifest, secret, pluginURL string, commandArgs
 		})
 	}
 
-	if m.RequestedPermissions.Contains(apps.PermissionActAsUser) {
-		elements = append(elements, model.DialogElement{
-			DisplayName: "Require user consent to use REST API first time they use the app:",
-			Name:        "consent",
-			Type:        "radio",
-			Default:     "require",
-			HelpText:    "please indicate if user consent is required to allow the app to act on their behalf",
-			Options: []*model.PostActionOptions{
-				{
-					Text:  "Require user consent",
-					Value: "require",
-				},
-				{
-					Text:  "Do not require user consent",
-					Value: "notrequire",
-				},
-			},
-		})
-	}
+	// if m.RequestedPermissions.Contains(apps.PermissionActAsUser) {
```

Comment:
maybe add a comment/give context wrt to the future plans for the commented-out code.

Reply:
Is there a ticket we can link?